client/components/data/query-reader-related-sites/index.jsx

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { requestRelatedSites } from 'calypso/state/reader/related-sites/actions';
+
+function QueryReaderRelatedSites( { tag = '', offset = 0 } ) {
+	const dispatch = useDispatch();
+
+	useEffect( () => {
+		dispatch( requestRelatedSites( { tag, offset } ) );
```

Comment:
I noticed that a similar component used an isFetching flag, I don't think it's neccessary here tbh but just thought I'd mention it https://github.com/Automattic/wp-calypso/blob/trunk/client/components/data/query-preferences/index.jsx#L7